index.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="favicon.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>[cCc] Nextdoor Nature Guides</title>
+    <title>[cCc] Nextdoor Nature Hub</title>
```

Comment:
Should this have been changed? It does seem inconsistent with everything else but is there a reason for that?